annif/openapi/annif.yaml

```diff
@@ -134,6 +134,41 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
       x-codegen-request-body-name: documents
+  /detect-language:
+    post:
+      tags:
+      - Languages detection
+      summary: detect the language of a text given a list of candidate languages
+      operationId: annif.rest.detect_language
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+              - text
+              - candidates
+              properties:
+                text:
+                  type: string
+                  description: input text
+                  example: A quick brown fox jumped over the lazy dog.
+                candidates:
+                  type: array
+                  description: candidate languages as ISO 639-1 codes
```

Comment:
Please change "ISO 639-1 codes" to "IETF BCP 47 codes", because not all supported languages are necessarily in ISO 639-1. (I noticed the same problem in Simplemma README and opened an [issue](https://github.com/adbar/simplemma/issues/30) there)